cou/zaza_utils/generic.py

```diff
@@ -66,21 +66,23 @@ def get_network_config(net_topology, ignore_env_vars=False, net_topology_file="n
     return net_info
 
 
-def get_pkg_version(application, pkg):
+def get_pkg_version(application, pkg, model_name=None):
```

Comment:
**nitpick:** 

I would like to know the type of the model_name from type annotation.